OktaIdxExample/OktaIdxExample/Signin/Signin.swift

```diff
@@ -53,6 +53,36 @@ public class Signin {
         viewController.present(navigationController, animated: true, completion: nil)
     }
     
+    internal func buttonTitle(for option: IDXClient.Remediation.Option?) -> String? {
+        guard let option = option else {
+            return "Restart"
+        }
+        
+        switch option.type {
+        case .identify:
+            return "Next"
+        
+        case .skip:
+            return "Skip"
+            
+        case .selectEnrollProfile: fallthrough
```

Comment:
What about `case .selectEnrollProfile, .enrollProfile`? It makes more explicit intention. 

It is tightly tied to the idea that after `selectEnrollProfile` is `entrollProfile`. I would use `fallthrough` when it doesn't matter what goes next.

Reply:
This is for the human-readable titles, so it's less about communicating the intent behind each remediation type, but about the messaging to convey to the user.